cmake/modules/platform/toolcfg/xlc.cmake

```
@@ -32,6 +32,11 @@ if(OMR_HOST_ARCH STREQUAL "ppc")
 
 	list(APPEND OMR_PLATFORM_CXX_COMPILE_OPTIONS -qlanglvl=extended0x)
 
+	if(NOT OMR_ENV_LITTLE_ENDIAN)
+		# add -qdebug=NETHUNK for BE platforms to C++ flags
```

I think it would be better to say what the flag does:

```suggestion
		# -qdebug=NETHUNK is an internal xlc flag resolving a compiler bug related to multiple inheritance
```